dspace/config/modules/usage-statistics.cfg

```diff
@@ -35,3 +35,7 @@ usage-statistics.authorization.admin.workflow=true
 # (see query.filter.* for query filter options)
 # Default value is true.
 #usage-statistics.logBots = true
+
+# Enable/disable if a matching for a bot should be case sensitive
+# Setting this value to true will increase cpu usage, but bots will be found more accurately
+usage-statistics.bots.case-insensitive=false
```

Comment:
In a minor release (e.g. 6.1), we should comment out this setting by default. In minor releases, our configs should be backwards compatible, so if this config is missing, nothing should "break".